nntrainer/utils/common_properties.h

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file common_properties.h
```

Comment:
as this file is intending to create properties for layers, how about placing it in layer folder?

Reply:
Done.